swift_algorithms.xcodeproj/project.pbxproj

```diff
@@ -935,7 +935,7 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
-				DEVELOPMENT_TEAM = 9L49SS68XP;
+				DEVELOPMENT_TEAM = SD6G3DF8CS;
 				INFOPLIST_FILE = swift_algorithms/Info.plist;
```

Comment:
@adincebic Would you mind reverting your project file change here?

Reply:
Thanks for your response. I reverted the project file.

Reply:
@thexande can we merge this now?